packages/xo-web/src/common/intl/messages.js

```diff
@@ -1285,6 +1285,9 @@ const messages = {
   spaceLeftTooltip: '{used}% used ({free} left)',
 
   // ----- New VM -----
+  createVmModalTitle: 'Create VM',
+  createVmModalWarningMessage:
+    'You will use more resources than you need. This will use the resources quota. Are you sure you want to continue?',
```

Comment:
Could we customize the message to let the user knew what is wrong with its configuration ?